src/OpenSpaceToolkit/Astrodynamics/Flight/Profile/Model/Tabulated.cpp

```diff
@@ -310,7 +308,7 @@ void Tabulated::setMembers(const Array<State>& aStateArray)
 {
     if (aStateArray.getSize() < 2)
     {
-        return;
+        throw ostk::core::error::RuntimeError("State array must have at least length 2.");
```

Comment:
This prevents us from creating an undefined tabulated model.

Reply:
Yeah that was my intention.

I don't really see why we want to keep creating things that are undefined, IMO it just creates more confusion.

The only use case I see for creating things that are undefined is to allow passing of a variable into a callback of some sort, but you can just as easily do it by initializing the variable and passing it